java/core/src/test/java/com/whylogs/api/resultsets/TestViewResultSet.java

```diff
@@ -0,0 +1,41 @@
+package com.whylogs.api.resultsets;
+
+import com.whylogs.api.logger.resultSets.ViewResultSet;
```

Comment:
consistency in casing for resultsets vs restultSets?

Reply:
haha good catch

Reply:
https://google.github.io/styleguide/javaguide.html#s5.2.1-package-names

It's all lower case